src/components/Chapter.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Chapter = () => (
+  <div className="chapter-info">
+    <div>
+      <p className="title position">Current Chapter</p>
+      <p className="chapter-name">Chapter 23</p>
+    </div>
+    <button type="button">Update Progress</button>
+  </div>
```

Comment:
- [ ] Please reduce the height and width of the `Update progress` button and make sure it matches with the zeplin design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185755178-e0866cf1-4a76-45c5-ae5a-e18e6ec76cbf.png)

### Zepline design

![image](https://user-images.githubusercontent.com/83803414/185755199-92800a1b-0646-450e-b76e-0439798f6064.png)